thiago_couto/class2/my_calc.rb

```diff
@@ -0,0 +1,40 @@
+puts "What would you like to do? ('add', 'subtract', 'multiply', 'divide')"
+opperation = gets.chomp
+
+def add(num1, num2)
+	answer = num1 + num2
+end
+def subtract(num1, num2)
+	answer = num1 - num2
+end
+def multiply(num1, num2)
+	answer = num1 * num2
+end
+def divide(num1, num2)
+	answer = num1 / num2
+end
+
+if opperation == "add" || opperation == "subtract" || opperation == "multiply" || opperation == "divide"
```

Comment:
Not bad! Maybe you could even wrap this in a loop so that just in case your user enters a wrong operation, they can try again!

```
loop do

  puts "What would you like to do? ('add', 'subtract', 'multiply', 'divide')"
  opperation = gets.chomp

  if !["add", "subtract", "multiply", "divide"].include?(opperation)
    puts "Sorry, incorrect operation"
    next
  end

  puts "What's the first number?"
  num1 = gets.chomp.to_i

  puts "what's the second number?"
  num2 = gets.chomp.to_i


  if opperation == "add"
    show_answer(add(num1,num2))
    break
  elsif opperation == "subtract"
    show_answer(subtract(num1,num2))
    break
  elsif opperation == "multiply"
    show_answer(multiply(num1,num2))
    break
  elsif opperation == "divide"
    show_answer(divide(num1,num2))
    break
  end
  
end
```